### ISSUE TITLE
Issue with @react-aria/utils : 3.23.2 Version Test cases failure.

### ISSUE THREAD

**Issue by balavarshan** [adobe/react-spectrum#6055]
### Provide a general summary of the issue here

I have upgraded @react-aria/utils : 3.23.0 to @react-aria/utils : 3.23.2 which shows the latest version. but facing test case failure issue with TypeError while running the test cases with @react-aria/utils : 3.23.2 version. 
Any thoughts on this.
![Screen Shot 2024-03-13 at 10 34 53 AM](https://github.com/adobe/react-spectrum/assets/163339167/0582667c-a6b4-47a7-8ace-bc62ec096f16)

### 🤔 Expected Behavior?

Test case should be success

### 😯 Current Behavior

Test case failure.

### 🖥️ Steps to Reproduce

Test case failure in local build

### Version

"@react/react-spectrum": "2.40.0"

### What browsers are you seeing the problem on?

Other

### If other, please specify.

Test case build failure.

### What operating system are you using?

macOS Monterey

### 🧢 Your Company/Team

Adobe

**Comment by LFDanLu** [comment]
Can you make sure you only have one version of the `@react-aria/utils` package installed? That `getOwnerDocument` should be a part of the `@react-aria/utils` package via its utilities

**Comment by balavarshan** [comment]
> Can you make sure you only have one version of the `@react-aria/utils` package installed? That `getOwnerDocument` should be a part of the `@react-aria/utils` package via its utilities

Yes i have only one version of `@react-aria/utils` package installed i have upgraded it to latest version.
![Screen Shot 2024-03-13 at 11 51 58 AM](https://github.com/adobe/react-spectrum/assets/163339167/0ac0a8d1-1fc4-4c5b-a30e-edbd589d3c01)

**Comment by LFDanLu** [comment]
If you pull in other packages that have their own dependencies on any react-aria packages it is possible to get duplicates even if you've specified a specific version in your `package.json`. Can you try the script listing in https://github.com/adobe/react-spectrum/wiki/Frequently-Asked-Questions-(FAQs)#why-are-there-errors-after-upgrading-a-react-spectrum-package to see if you have any dupes?

**Comment by balavarshan** [comment]
> If you pull in other packages that have their own dependencies on any react-aria packages it is possible to get duplicates even if you've specified a specific version in your `package.json`. Can you try the script listing in https://github.com/adobe/react-spectrum/wiki/Frequently-Asked-Questions-(FAQs)#why-are-there-errors-after-upgrading-a-react-spectrum-package to see if you have any dupes?


I see this when i run the script. any inputs.
![Screen Shot 2024-03-13 at 1 11 31 PM](https://github.com/adobe/react-spectrum/assets/163339167/7c1f826f-ffa5-4dfb-bcf3-248a0864b6fe)

**Comment by balavarshan** [comment]
> > If you pull in other packages that have their own dependencies on any react-aria packages it is possible to get duplicates even if you've specified a specific version in your `package.json`. Can you try the script listing in https://github.com/adobe/react-spectrum/wiki/Frequently-Asked-Questions-(FAQs)#why-are-there-errors-after-upgrading-a-react-spectrum-package to see if you have any dupes?
> 
> I see this when i run the script. ![Screen Shot 2024-03-13 at 1 11 31 PM](https://private-user-images.githubusercontent.com/163339167/312607768-7c1f826f-ffa5-4dfb-bcf3-248a0864b6fe.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTAzNjIyNjIsIm5iZiI6MTcxMDM2MTk2MiwicGF0aCI6Ii8xNjMzMzkxNjcvMzEyNjA3NzY4LTdjMWY4MjZmLWZmYTUtNGRmYi1iY2YzLTI0OGEwODY0YjZmZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzEzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxM1QyMDMyNDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YTI0N2NlODAxNmU3ZWM4ZGYxMzNhNzhhYjRmZGM5NDM1YzI3ZjVlMTk2MzZhNzBjZjA2N2JkNGJjNjA1OTY2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.m45PYDtlN6LBNj_o6DeiDmYuG8K5Br9ULRo7A_U_NwU)

I resolved all the dupes but still the tests are failing.
![Screen Shot 2024-03-13 at 1 52 49 PM](https://github.com/adobe/react-spectrum/assets/163339167/e76fe8eb-d9fa-4518-8a77-2b75ba0f9f53)

**Comment by balavarshan** [comment]
> If you pull in other packages that have their own dependencies on any react-aria packages it is possible to get duplicates even if you've specified a specific version in your `package.json`. Can you try the script listing in https://github.com/adobe/react-spectrum/wiki/Frequently-Asked-Questions-(FAQs)#why-are-there-errors-after-upgrading-a-react-spectrum-package to see if you have any dupes?


I resolved all the dupes but still the tests are failing.
![Screen Shot 2024-03-13 at 1 52 49 PM](https://github.com/adobe/react-spectrum/assets/163339167/e76fe8eb-d9fa-4518-8a77-2b75ba0f9f53)

**Comment by LFDanLu** [comment]
strange, not sure what is happening here then without a base reproduction. If you dig into your `node_modules` are you able to find the `getOwnerDocument` in the `react-aria/utils` dist?

**Comment by balavarshan** [comment]
> strange, not sure what is happening here then without a base reproduction. If you dig into your `node_modules` are you able to find the `getOwnerDocument` in the `react-aria/utils` dist?

No, i couldn't find anything in the whole repo
![Screen Shot 2024-03-13 at 2 19 52 PM](https://github.com/adobe/react-spectrum/assets/163339167/fd70560e-00a7-4b6e-96ed-b67a96fbf862)

**Comment by LFDanLu** [comment]
Your VSCode search might has `node_modules` automatically excluded from its search results, could you check? It will also be easier for us to debug this if you are able to create a base reproduction test. You can probably just write a test that just renders a Dialog and see if that fails in a fresh codesandbox/stackblitz. If it does, then that will give the team somewhere to dig but if not then there is probably something app specific happening here

**Comment by snowystinger** [comment]
Closing for now. If we can get a reproduction, we can reopen.
